### ISSUE TITLE
Game "Freezes" When Walking Around In Town/Cities

### ISSUE THREAD

**Issue by Cpl-Jordan** [CleverRaven/Cataclysm-DDA#47175]
# Describe the bug

When playing Cataclysm and you walk around in any town or city the game freezes for around 2-5 seconds.

# Steps To Reproduce

(Method 1) 
1.) Move your Character into any Town or City.
2.) Move your Character in a straight line down a roadway.
3.) See that the game freezes when your character moves between street lamps. (See Screenshot 1)

(Method 2)
1.) Move your Character into any Town or City.
2.) Move your character onto any roadway.
3.) Walk across the roadway and pass the yellow road pavement and see the game freeze for 2-5 seconds. (See Screenshot 2)

# Expected behavior
The game should run smoothly when moving around any town/cities.

# Screenshots
(Screenshot For Method 1)
![Cataclsym_Freezes_Lamps](https://user-images.githubusercontent.com/77216510/106493061-a9143080-6486-11eb-8e9c-0ba7fbf4e205.png)

(Screenshot For Method 2)
![Cataclsym_Freezes_Yellow](https://user-images.githubusercontent.com/77216510/106493201-d4971b00-6486-11eb-8940-0de893ddd181.png)

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 2004
- Game Version: 0.E-9338-gf1a1af3-dirty [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

# Additional context
I have opened an issue like this in the past but it looks like the issue still exists. This was my previous post #46653

Save File 2 is in a spot where if you move the character to the right down the street you can see the game freezes longer then the 2-5 seconds when walking around the city.

Save File 1.) [Save_1.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5905594/Save_1.zip)

Save File 2.) [Save_2.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5905619/Save_2.zip)

**Comment by ZhilkinSerg** [comment]
There are just a lot of items I guess.

**Comment by Salty-Panda** [comment]
Is it some existing run or a fresh save?
If the former then it's a matter of loading tons of items, already reported a few times.

**Comment by Cpl-Jordan** [comment]
This is an existing run. But yeah its the game loading a ton of items.

**Comment by Aivean** [comment]
This should be addressed by  #46837 and #46763. Unfortunately, these fixes apply only to the newly saved submaps, so please try to reproduce this issue in the freshly generated city.

**Comment by Aivean** [comment]
I tried to reproduce the issue on "save_2". First I walked around and used "quicksave" to ensure the submaps are saved with compressed items. Then I reloaded the game and walked around profiling the submap loading times:

![image](https://user-images.githubusercontent.com/2865203/107305171-4fd47000-6a37-11eb-9dbf-a58fd52ff8e6.png)

On my 2016 macbook the lag is barely perceptible and profiler tells it's around ≈117ms. 

Untill there is hard evidence that this issue is reproducible I'm closing it.
